test/e2e-upgrade-tests.sh

```diff
@@ -90,6 +90,10 @@ header "Running postupgrade tests"
 go_test_e2e -tags=postupgrade -timeout=${TIMEOUT} ./test/upgrade \
   --resolvabledomain=$(use_resolvable_domain) || fail_test
 
+header "Running postdowngrade tests"
```

Comment:
Do you want to "install_latest_release" before calling this test? I think this is inteneded, looking at the readme below.